README.md

```diff
@@ -5,24 +5,17 @@
 
```

Comment:
Shouldn't we also get rid of these links? As only `jpscore` still exists, but the documentation is heavily updated.